packages/react-app/src/views/CreateZone.jsx

```diff
@@ -49,8 +48,8 @@ function Polygons({ writeContracts, address, injectedProvider, readContracts, us
         scrollWheelZoom={false}
       >
         {/* TODO: Implement dark mode https://stamen-tiles.a.ssl.fastly.net/toner/{z}/{x}/{y}.png not working */}
-        <TileLayer attribution="" url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
-        <DrawTools />
+        <TileLayer attribution="OpenStreetMap" url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
+        <ControlTools selectedNetwork={selectedNetwork} />
```

Comment:
Nice! So basically, when we create a new zone, we open a modal in front of the main content with a map of the new zone? Instead of just sending an alert saying we made a zone. 

Is that right?

Reply:
We show a box with the data and buttons to share or verify the zone you just created. 
We also show a toast stating the transaction was succesful.